### ISSUE TITLE
Gebruik van text-overflow ellipsis minimaliseren

### ISSUE THREAD

**Issue by bramhoosemans** [dso-toolkit/dso-toolkit#774]
In het voorbeeld de modal wordt op de heading een text-overflow: ellipsis; gezet. Dit is niet wenselijk. De toolkit moet nagelopen worden om ellipsis overal te verwijderen.

**Comment by daverhelst** [comment]
Gezocht op "text-overflow: ellipsis" in node_modules map, met de volgende resultaten:

- \node_modules\@frctl\mandelbrot\assets\scss\components\_header.scss - _header.scss
```
.Header {
    display: flex;
    align-items: center;
    position: relative;
    z-index: 10;
    background-color: $color-header-background;
    box-shadow: inset 0 -1px 0 0 rgba(black, 0.25), 0 2px 0 rgba(black, 0.1);
    text-overflow: ellipsis;
    color: $color-header-content;
}

.Header-title {
    @include link-default(inherit, none);
    padding: 0.75rem 1rem;
    font-size: 1.125rem; // 18px
    line-height: (24 / 18);
    white-space: nowrap;
    overflow: hidden;
    text-overflow: ellipsis;
}
```

**Comment by gofix** [comment]
En nu, @daverhelst ? Zouden we hetgeen gebeurt op `dt` in DSO weer moeten tegenstylen? Lijkt me wel toch?

**Comment by daverhelst** [comment]
Aanvullen van sass linter om "text-overflow: ellipsis" als error terug te laten komen, zodat er bewust een keuze gemaakt moet worden om deze error uit te zetten, of de regel te verwijderen.

**Comment by daverhelst** [comment]
Na onderzoek is gebleken dat er geen optie bestaat om een bepaalde css-property als error te bestempelen. Andere linters zoals https://stylelint.io/ kunnen dit wel

**Comment by daverhelst** [comment]
@gofix op je eerdere vraag of we dit voor de `dt` moeten tegen stijlen. Als dat wenselijk is wel, maar dan wel in een ander issue lijkt me. Deze gaat over de header in de modal (met een klein uitstapje naar ellipsis in dso.css)

**Comment by daverhelst** [comment]
Het vervangen van sass-lint door Stylelint zal in issue #805 opgepakt worden

**Comment by gofix** [comment]
> @gofix op je eerdere vraag of we dit voor de `dt` moeten tegen stijlen. Als dat wenselijk is wel, maar dan wel in een ander issue lijkt me. Deze gaat over de header in de modal (met een klein uitstapje naar ellipsis in dso.css)

Dit ticket gaat over alle voorkomende `text-overflow:ellipsis` toch? Ook als dit uit bootstrap komt. En daarom mijn vraag: aangezien bootstrap dus (via de mixin `@include text-overflow;`) de `dt` van ellipsis voorziet: moeten we dat tegenstijlen. Of, zit dit alleen in de `.dl-horizontal` die we in DSO Toolkit toch niet ondersteunen. Als dat laatste het geval is, is dit issue klaar!

**Comment by daverhelst** [comment]
Ja
